docs/content/reference/migration/migration-0-16.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Migrating from 0.15 to 0.16
+order: 160
+---
+
+
+## `timeless` replaced by `static`
+The concept of _timeless_ data has been replaced by _static_ data.
+Except for the name change, it is mostly the same.
+
+Static data is data that shows up at all times, on all timelines.
+
+In 0.15 you could log a component as timeless once, and then change it to be temporal (non-timeless) later (or the other way around). This is no longer allowed for static data.
+Now, once you log something (a specific component on a specific entity) as static, it must stay static forever.
+Conversely, if you log something as temporal (non-static), it must not be logged as static in the future.
```

Comment:
I understood it as that if you log temporal first and then log static, the component becomes static ignoring what was logged temporally. Is that incorrect?

Reply:
:point_up: that's correct -- I'll update.